modules/mono/mono_gd/gd_mono.cpp

```diff
@@ -260,32 +260,38 @@ using godot_plugins_initialize_fn = bool (*)(void *, GDMonoCache::ManagedCallbac
 #endif
 
 #ifdef TOOLS_ENABLED
-godot_plugins_initialize_fn initialize_hostfxr_and_godot_plugins(bool &r_runtime_initialized) {
-	godot_plugins_initialize_fn godot_plugins_initialize = nullptr;
-
-	HostFxrCharString godot_plugins_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.dll"));
 
-	HostFxrCharString config_path = str_to_hostfxr(
-			GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json"));
+load_assembly_and_get_function_pointer_fn initialize_hostfxr(bool &r_runtime_initialized) {
+	String api_assemblies_config = GodotSharpDirs::get_api_assemblies_dir().path_join("GodotPlugins.runtimeconfig.json");
+	if (!FileAccess::exists(api_assemblies_config)) {
+		WARN_PRINT("Mono glue not found. Consider running the next commands from the Godot source root and recompiling using `mono_glue=yes`:\n\
+<godot_binary> --generate-mono-glue ./modules/mono/glue\n\
+./modules/mono/build_scripts/build_assemblies.py --godot-output-dir ./bin");
+		ERR_FAIL_V_MSG(nullptr, "Could not initialize hostfxr.");
+	}
 
-	load_assembly_and_get_function_pointer_fn load_assembly_and_get_function_pointer =
+	HostFxrCharString config_path = str_to_hostfxr(api_assemblies_config);
+	load_assembly_and_get_function_pointer_fn setup_assembly =
 			initialize_hostfxr_for_config(get_data(config_path));
-	ERR_FAIL_NULL_V(load_assembly_and_get_function_pointer, nullptr);
+	ERR_FAIL_NULL_V(setup_assembly, nullptr);
```

Comment:
I think the shorter name here is more confusing than before, and I think we were being explicit on purpose to match the name of the function in hostfxr.

Reply:
The original name was the same as the type, which is very long and verbose. Since the entire function can fit on an ISO/ANSI screen with 24 lines, the type is always visible, and so the variable's name is not important enough to warrant 38 characters, almost half of the 80 characters width of an ISO/ANSI screen.

But it's just a variable name with a verbose type in a short function. I am not going to defend this change much.

Reply:
The old name is much better in this case, as it describes what the function does. I wouldn't be able to tell what `setup_assembly` means without looking at the type (supposing that it's using the typedef). I blame the long and verbose name on the .NET hosting API, which we don't have control over.